src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -311,8 +309,24 @@ class TinyMCEConfig extends HTMLEditorConfig implements i18nEntityProvider
         'promotion' => false,
         'upload_folder_id' => null, // Set folder ID for insert media dialog
         'link_default_target' => '_blank', // https://www.tiny.cloud/docs/tinymce/6/autolink/#example-using-link_default_target
+        // Default set of valid_elements which apply for all new configurations
+        'valid_elements' => "@[id|class|style|title],a[id|rel|rev|dir|tabindex|accesskey|type|name|href|target|title"
+        . "|class],-strong/-b[class],-em/-i[class],-strike[class],-u[class],#p[id|dir|class|align|style],-ol[class],"
+        . "-ul[class],-li[class],br,img[id|dir|longdesc|usemap|class|src|border|alt=|title|hspace|vspace|width|height|align|name|data*],"
```

Comment:
```suggestion
        . "-ul[class],-li[class],br,img[id|dir|longdesc|usemap|class|src|border|alt|title|hspace|vspace|width|height|align|name|data*],"
```
Is that a typo with the = for alt?

Do we want name, hspace, space for the old extended_valid_elements from admin that's being deleted?

Reply:
> Is that a typo with the = for alt?

The `=` is correct, as per https://www.tiny.cloud/docs/tinymce/latest/content-filtering/#valid_elements
Basically, it's defaulting to including a blank `alt` property for all images which don't have a specific alt text defined for them, which is correct for accessibility.

> Do we want name, hspace, space for the old extended_valid_elements from admin that's being deleted?

It's only being deleted because I've moved those here. They seem safe, so I'd be inclined to keep them here. If you have a specific reason not to, I can move them back to `extended_valid_elements` in the "cms" config where they have previously lived.

Reply:
> Do we want name, hspace, space for the old extended_valid_elements from admin that's being deleted?

Sorry I'm blind, somehow I managed to not noticed they're copied over. Yes keep them as is in this PR